source/_posts/2021-12-11-release-202112.markdown

```diff
@@ -776,6 +780,26 @@ dedicated `number` entities.
 
 {% enddetails %}
 
+{% details "Statistics" %}
+
+All statistical characteristics were removed from sensor attributes and are
+now selectable through `state_characteristic` as the state of the sensor
+directly.
+The parameter `state_characteristic` will become mandatory for configuration
+in a future release.
+
+Users will need to adapt their configuration and expose the desired characteristic.
```

Comment:
Don't refer to the reader as "users". Use "you" instead.

Reply:
Done